my_game.py

```diff
@@ -556,6 +588,13 @@ def on_key_press(self, key: int, modifiers: int):
         game_view = GameView()
         self.window.show_view(game_view)
 
+    def on_click_start(self, event):
+        """
+        Start the game when start button is clicked
+        """
+        game_view = GameView()
```

Comment:
Please make a _game_start()_ function instead of duplicating code in _on_key_press()_ and _on_click_start()_

Reply:
You want a class or a def?